freeipa/src/main/java/com/sequenceiq/freeipa/entity/ImageEntity.java

```diff
@@ -32,8 +39,14 @@ public class ImageEntity {
 
     private String imageName;
 
+    // Deprecated because this is stored as unecrypted data
     private String userdata;
 
+    // The new userdata field which stored in vault
```

Comment:
please drop it

Reply:
I think this is a useful information

Reply:
yes, and this is already present in the name and in the type of the field. every secret is in vault, and the field name contains userdata. so this is just the duplication of the same